lib/Doctrine/Common/DataFixtures/Executor/AbstractExecutor.php

```diff
@@ -144,10 +144,11 @@ public function purge()
     /**
      * Executes the given array of data fixtures.
      *
-     * @param array $fixtures Array of fixtures to execute.
-     * @param bool  $append   Whether to append the data fixtures or purge the database before loading.
+     * @param FixtureInterface[] $fixtures          Array of fixtures to execute.
+     * @param bool               $append            Whether to append the data fixtures or purge the database before loading.
+     * @param bool               $singleTransaction Whether to use a single transaction when loading fixtures.
      */
-    abstract public function execute(array $fixtures, $append = false);
+    abstract public function execute(array $fixtures, $append = false /* bool $singleTransaction = false */);
```

Comment:
instead of adding a new argument that is relevant only for the ORM, maybe this should be a constructor argument of the ORMExecutor instead.